contributor-guide/modules/ROOT/pages/testing/boost-test-matrix.adoc

```diff
@@ -1,286 +1,178 @@
 = Boost Test Matrix
-:idprefix:
-:idseparator: -
+:navtitle: Boost Test Matrix
 
-*Legacy content, currently being updated.*
+It's straightforward to run regression tests on your Boost clone, by utilizing some scripts available for the purpose.
 
-Running Regression Tests Locally
---------------------------------
-
-***It's easy to run regression tests on your Boost
- clone.***
-
-
-To run a library's regression tests, run Boost's
- [b2](/build/) utility from the
- *<boost-root>*/libs/*<library>*/test directory. To run a
- single test, specify its name (as found in 
- *<boost-root>*/libs/*<library>*/test/Jamfile.v2) on the
- command line.
-
-
-See the [Getting
- Started guide](/doc/libs/release/more/getting_started/index.html) for help building or downloading
- bjam for your platform, and navigating your Boost
- distribution.
-
-
-To run every library's regression tests, run b2
- from *<boost-root>*/status directory.
-
-
-To run B2's regression tests, run "python
- test\_all.py" from *<boost-root>*/tools/build/v2/test
- directory. (Python 2.3 ≤ version < 3.0 required.)
-
-Running Boost's Automated Regression and Reporting
---------------------------------------------------
-
-This runs all Boost regression tests and reports the results back to
- the Boost community.
-
-
-### Requirements
+== Script Requirements
 
+To run the python based scripts, ensure you have:
 
 * Python (2.3 ≤ version < 3.0).
```

Comment:
Python is not a requirement for many of these things. Neither is git once you have the sources. If you don't have the source files yet, then the reader shouldn't be here because this is deep into the contributor guide.